README.md

```diff
@@ -211,7 +211,7 @@ If this doesn't work, consider asking for help at [Winget's project page](https:
 
 A: This is a known limitation of Winget. 
 
-See more details on the issue https://github.com/microsoft/winget-cli/issues/2603.<br>
+Please take a look at more details on the issue https://github.com/microsoft/winget-cli/issues/2603.<br>
```

Comment:
Feels completely unnecessary to me as a reader. What is the reasoning for this suggestion?

Reply:
It is more pleasing.

Reply:
You're relying on a tool and not on your skill. The tool you are using, might be able to identify gramatical issues, but does not provide good suggestions.

If you want to change the sentance, "Please take a look at more details..." is not a good English phrasing. It is cumbersome.

An alternative phrasing might be: "For more details, see this issue"